### PR TITLE
Minor improvements

### PR DESCRIPTION
**Pull request by sindresorhus**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,17 +74,20 @@ const dropbox = {
...
-      title: 'Directory to save the files in',
+      title: 'Upload Directory',
+      description: 'The directory to save the files in.',
       type: 'string',
-      pattern: '^\/(.|[\r\n])*[^\/]$'
+      pattern: /^\/(.|[\r\n])*[^/]$/.source,
     },
     autorename: {
-      title: 'Automatically rename to avoid conflicts',
+      title: 'Auto-Rename',
```
Comment (sindresorhus) [inline index.js:83]: Alternatively:
```suggestion
      title: 'Avoid Conflicts',
```

**Comment by sindresorhus** [comment]
Some more feedback (things I don't intend to solve in this PR)

---

Is there really no better API way to check if it was a file conflicts?

https://github.com/karaggeorge/kap-dropbox/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR65

---

I think you should include the filename in the error here: https://github.com/karaggeorge/kap-dropbox/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR66

**Comment by sindresorhus** [comment]
I think you should include the filename in the error here: https://github.com/karaggeorge/kap-dropbox/pull/2/files#diff-168726dbe96b3ce427e7fedce31bb0bcR66

**Comment by karaggeorge** [comment]
> Is there really no better API way to check if it was a file conflicts?

I'm not really checking for conflicts, since there's no way currently to show feedback and restart the process with some sort of UI.

The autoRename is a feature available through Dropbox's upload endpoint, so I figured I'd forward it to the plugin config. I'm not doing any of the naming myself

**Comment by sindresorhus** [comment]
> The autoRename is a feature available through Dropbox's upload endpoint, so I figured I'd forward it to the plugin config. I'm not doing any of the naming myself

Yeah, I just meant that it's weird that we don't get any JSON response we can check instead.

File: index.js [hunk 2]
```diff
@@ -73,17 +74,20 @@ const dropbox = {
   action,
   config: {
     path: {
-      title: 'Directory to save the files in',
+      title: 'Upload Directory',
+      description: 'The directory to save the files in.',
       type: 'string',
-      pattern: '^\/(.|[\r\n])*[^\/]$'
+      pattern: /^\/(.|[\r\n])*[^/]$/.source,
```
Comment (sindresorhus) [inline index.js:80]: Why are you allowing newlines here?

Comment (karaggeorge) [reply index.js:80]: I'm not sure where I got that regex from. I haven't look at this code in a long time. I mostly updated the auth stuff. I can take a look and make it a bit more strict

**Pull request merged**